rust/cardano-chain-follower/src/mithril_snapshot_iterator.rs

```diff
@@ -196,7 +197,17 @@ impl MithrilSnapshotIterator {
         let res = task::spawn_blocking(move || {
             #[allow(clippy::unwrap_used)] // Unwrap is safe here because the lock can't be poisoned.
             let mut inner_iterator = inner.lock().unwrap();
-            inner_iterator.next()
+            stats::start_thread(
```

Comment:
This doesn't need an independent statistic, it's not really a task or a worker, its just a way to use a synchronous iterator inside an asynchronous function.